kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/BaseContentService.java

```diff
@@ -4339,6 +4339,20 @@ protected void removeResource(ContentResourceEdit edit, boolean removeContent) t
 		// htripath -store the metadata information into a delete table
 		// assumed uuid is not null as checkExplicitLock(id) throws exception when null
 
+
+		//check if resource is of type CitationsList and clean up citation tables
+		if (removeContent) {
+			if( edit.getResourceType().equals(CITATIONS_RESOURCE_TYPE_ID)) {
```

Comment:
Could the conditions be chained in one `if`?

Reply:
done